### PR TITLE
javascript-custom-methods exercise - adrian arias

### PR DESCRIPTION
**Pull request by adrianm-arias**
Screen Shot 2022-07-25 at 10 33 29 AM

### CODE REVIEW

File: javascript-custom-methods/calculator.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      finalCount += numbers[i];
+    }
+    return finalCount;
+  },
+  getAverage: function (numbers) {
+    var finalCount = 0;
+    for (var i = 0; i < numbers.length; i++) {
+      finalCount += numbers[i];
+    }
+    var Average = finalCount / numbers.length;
```
Comment (Zhiwutian) [inline javascript-custom-methods/calculator.js:30]: make sure to use a lowercase variable!

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Great work! Just an uppercase variable to remove!

File: javascript-custom-methods/calculator.js (hunk 1 above)
Comment (adrianm-arias) [reply javascript-custom-methods/calculator.js]: done, sorry!

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**